.circleci/config/executors/@executors.yml

```diff
@@ -26,7 +26,7 @@ alpine:
 docker-env-go-test-remote-docker:
   resource_class: medium
   docker:
-    - image: "docker.mirror.hashicorp.services/circleci/golang:1.15.4-buster"
```

Comment:
did you do these manually or did it happen automatically?

i seem to remember it not happening automatically, and doing it manually broke tests for me